### PR TITLE
Approximate Arcs With Beziers

### LINKED ISSUE #103: Convert arc to cubic bezier
How to convert arc to cubic bezier with this tool?

### PR DESCRIPTION
**Pull request by tatarize**
Closes #103
Replaces #116 

Adds in the ability to replace all Arcs within a Path with either Cubic or Quadratic bezier curves. Adds in the ability to perform this action on on the Arc line segments themselves.

### CODE REVIEW

File: svgpathtools/path.py [hunk 1]
```diff
@@ -2906,6 +2986,32 @@ def is_contained_by(self, other):
...
+        for s in range(len(self)-1, -1, -1):
+            segment = self[s]
+            if not isinstance(segment, Arc):
+                continue
+            arc_required = int(ceil(abs(segment.delta) / sweep_limit))
+            self[s:s+1] = list(segment.as_cubic_curves(arc_required))
+
+    def approximate_arcs_with_quads(self, error=0.1):
+        """
+        Iterates through this path and replaces any Arcs with cubic bezier curves.
```
Comment (njhurst) [inline svgpathtools/path.py:3004]: quadratic

**Review by njhurst — APPROVED** [review]

**Comment by tatarize** [comment]
Added tests for the code. I'm kinda shocked how much better Cubic is than Quad. But I guess it makes a lot of sense.
@mathandy I'd be surprised if there was any objection to this code.

**Comment by njhurst** [comment]
The arc length test was set with a really poor target precision so that it would run quickly, I couldn't find it any more, but you should update it to the same precision as the other cases with this change.  Cubic beziers converge as O(e^6), Quadratic as O(e^4), so cubics quickly dominate.  For geometric approximation it's even more stark, but I don't recall the convergence rates.

**Comment by mathandy** [comment]
Thanks

**Pull request merged**